styles/widgets/generic/treeView.generic.less

```diff
@@ -31,6 +31,8 @@
     @GENERIC_SELECT_ALL_TEXT_OFFSET: 31px;
 
     @GENERIC_TREEVIEW_ARROW_ICON_SIZE: 22px;
+
+    @GENERIC_TREEVIEW_SEARCH_EDITOR_MARGIN: 5px;
```

Comment:
Make this name consistent with `GENERIC_LIST_SEARCHBOX_MARGIN_BOTTOM`

Reply:
Done